services/dynamic-sidecar/requirements/_test.in

```diff
@@ -10,6 +10,7 @@ docker
 faker
 flaky
 pytest
+pytest-aiohttp
```

Comment:
This is deprecated package why do you need this?

Reply:
This is required for a fixture used in a test. The code for the notifier (python-socketio test)

Reply:
can you please point me there? I would like to understand it. pytest-aiohttp latest version is basically pytest-asyncio with some different names...

Reply:
OK, I managed to fix it and use pytest-asyncio